### PR TITLE
Add IndexHandler, config file, custom message

### PR DESCRIPTION
**Pull request by C0RB3N**

### CODE REVIEW

File: experimentalServer/experimentalServer.go [hunk 1]
```diff
@@ -25,7 +28,16 @@ type ResponseJSON struct {
...
 }
-
+// load config
+func GetMeta(path string) (MetaConfig *MetaJSON) {
+	MetaConfig = new(MetaJSON)
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		log.Fatalf("Error opening config file: %v", err)
+	}
+	json.Unmarshal(data, &MetaConfig)
```
Comment (erbbysam) [inline experimentalServer/experimentalServer.go]: If you don't mind, can you check the return on `json.Unmarshal` and return a fatal error if it fails?
https://golang.org/pkg/encoding/json/#Unmarshal

File: experimentalServer/config.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+"Message": "If you would like to support this instance, consider donating!",
```
Comment (erbbysam) [inline experimentalServer/config.json]: Can you leave the `Message` field blank?

I don't expect anyone to donate to me for this project/hosting, so I would prefer if this was not the default value :)

File: experimentalServer/experimentalServer.go (hunk 1 above)
Comment (C0RB3N) [reply experimentalServer/experimentalServer.go]: Yep! I thought I had done that 😂

**Comment by C0RB3N** [comment]
Fixed!

**Pull request merged**